src/olympia/addons/models.py

```diff
@@ -499,6 +499,8 @@ def initialize_addon_from_upload(cls, data, upload, is_listed=True):
         if generate_guid:
             data['guid'] = guid = generate_addon_guid()
 
+        data = cls.resolve_webext_translations(data, upload)
```

Comment:
Though I guess this does not matter since you have the default locale from the package...